### ISSUE TITLE
这些模型的训练数据时间点截止哪一天？

### ISSUE THREAD

**Issue by 13255944044** [shiyu-coder/Kronos#117]
Model	Tokenizer	Context length	Params	Open-source
Kronos-mini	[Kronos-Tokenizer-2k](https://huggingface.co/NeoQuasar/Kronos-Tokenizer-2k)	2048	4.1M	✅ [NeoQuasar/Kronos-mini](https://huggingface.co/NeoQuasar/Kronos-mini)
Kronos-small	[Kronos-Tokenizer-base](https://huggingface.co/NeoQuasar/Kronos-Tokenizer-base)	512	24.7M	✅ [NeoQuasar/Kronos-small](https://huggingface.co/NeoQuasar/Kronos-small)
Kronos-base	[Kronos-Tokenizer-base](https://huggingface.co/NeoQuasar/Kronos-Tokenizer-base)	512	102.3M	✅ [NeoQuasar/Kronos-base](https://huggingface.co/NeoQuasar/Kronos-base)
Kronos-large	[Kronos-Tokenizer-base](https://huggingface.co/NeoQuasar/Kronos-Tokenizer-base)	512	499.2M	❌

**Comment by shiyu-coder** [comment]
The cut-off date for the training set is February 2024 (inclusive). The validation set spans from March to May 2024 (inclusive), and the out-of-sample period begins in June 2024.

**Comment by 13255944044** [comment]
多谢
